src/dropdowns/dropdown-trigger.spec.ts

```diff
@@ -0,0 +1,99 @@
+import {it, describe, expect, injectAsync, TestComponentBuilder} from 'angular2/testing';
+import {Component} from 'angular2/core';
+import {NglDropdown} from './dropdown';
+import {NglDropdownTrigger} from './dropdown-trigger';
+import {dispatchKeyEvent} from '../../test/helpers';
+
+function getDropdownTrigger(fixtureElement: HTMLElement): HTMLElement {
+  return <HTMLElement>fixtureElement.childNodes[0].childNodes[0];
+}
+
+describe('`nglDropdownTrigger`', () => {
+
+  it('should open the dropdown when it is clicked', testAsync(({fixture, done}) => {
+    const dropdownTrigger = getDropdownTrigger(fixture.nativeElement);
+    fixture.detectChanges();
+
+    spyOn(fixture.componentInstance, 'setOpen').and.callFake((isOpen: boolean) => {
+      expect(isOpen).toBe(true);
+
+      done();
+    });
+
+    dropdownTrigger.click();
+    fixture.detectChanges();
```

Comment:
Since this is synchronous now, you can write something like:
```js
    dropdownTrigger.click();
    expect(fixture.componentInstance.setOpen).toHaveBeenCalledWith(true);

    fixture.componentInstance.open = true;
    fixture.detectChanges();
    dropdownTrigger.click();
    expect(fixture.componentInstance.setOpen).toHaveBeenCalledWith(false);
```